src/build/index.mjs

```diff
@@ -0,0 +1,8 @@
+export * from './get_build_environment.mjs';
```

Comment:
Could you clarify the philosophy behind this module?

There seems to be things at different levels here.
run_action and get_root_dir are cornerstones of our build system, being here seems right. You can imagine us reusing that in outline-server, for instance.

get_build_.number|parameters is very tied to our application. they should probably be where the application code is.

spawn_stream looks like a general-use utility. It should probably be under infrastructure. Perhaps run_webpack too.

In a way, general build utilities could be under infrastructure as well.  So run_action and get_root_dir could be in infrastructure/build

I'm not sure what to do with get_webpack_build_mode. But it depends on a build mode concept which is specific to our application, so it probably belongs with the application code.

Reply:
Ah, no philosophy, just trying to start to move toward a monorepo. We can discuss how to better split this up first, instead. I'll include that in my doc summarizing the proposed changes #1468